jsx/FilterableDataTable.js

```diff
@@ -36,6 +36,20 @@ class FilterableDataTable extends Component {
     this.setState({filter});
   }
 
+  /**
+   * Parse filter to remove filter properties when opposite is set to true
+   *
+   * @param {object} filter passed from FilterForm
+   * @return {object} filter
+   */
+  parseFilter(filter) {
+    const parsedFilter = {};
+    for (const [filterId, filterProps] of Object.entries(filter)) {
+      if (filterProps.opposite === undefined || !filterProps.opposite) parsedFilter[filterId] = filterProps;
```

Comment:
nice. 
* any other conditions/operators besides opposite that should be included? 
* If i'm reading this right, isn't there a case where the empty const `{}` would be returned?

Reply:
That's right, an empty object can be returned. I tested that case and it seemed to be fine.
There is no other conditions we can include that I noticed, but maybe Henri will have suggestions.

Reply:
This seems like a good way of removing the value from displaying in the filter. If we want to push this through quickly, I think this works. 

However, my main concern with removing the value from the filter is that it obfuscates what's being filtered, and I generally don't think we should be hiding from the user what's being filtered. I'm also concerned about building more code infrastructure around the single-use `opposite` paradigm, which would need an overhaul when we properly implement excludes functionality. Preferably, a structure where users can choose to exclude certain values from their filter would be implemented, which would essentially mean removing this code in the long term.

In summary, I think this patch does what it's intended to do well, but I'm worried it moves the filtering in an alternative direction to what we should be moving towards to be handling this sort of functionality.

Reply:
Would it be better just to change the element to a multi-select?  no obfuscation required.

Reply:
Sure, I can do that. The filter will still need to be modified for the complement field set.

Reply:
Yes, making it a multi-select and have it auto-select all the values except for "close" could definitely work!

Reply:
ok.  (inhale) let's do that.  i'll ping the team channel and otherwise @laemtl go for it.